pom.xml

```diff
@@ -5,18 +5,18 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.iangabrielsanchez</groupId>
-    <artifactId>Bloop-API</artifactId>
+    <artifactId>bloop</artifactId>
     <packaging>pom</packaging>
     <version>1.0.0-SNAPSHOT</version>
     <modules>
-        <module>listener-service</module>
-        <module>api-core</module>
-        <module>listener-service-gui</module>
+        <module>bloop-api</module>
```

Comment:
@iangabrielsanchez Do we merge this into Develop? If so, how do we do that since folders are named differently? Merge, then delete?